src/smdba/postgresqlgate.py

```diff
@@ -155,9 +155,7 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
```

Comment:
Ok, it seems for older postgresql version (lower than 9.6) increasing checkpoint_segments to 8 sounds reasonable.